doc/whats_new.rst

```diff
@@ -35,6 +35,8 @@ Changelog
 
 - Add `trig_shift_by_type` parameter in :func:`mne.io.read_raw_brainvision` to allow to specify offsets for arbitrary marker types by `Henrich Kolkhorst`_
 
+- Refactor code within :func:`make_dics` and :func:`make_lcmv` into :func:`_compute_beamformer` by `Britta Westner`_
```

Comment:
`mne.beamformer.make_dics` or it won't link properly (this is how it's nested in `python_reference.rst`, same with `make_lcmv`. And there is no need to reference private functions for users (also it won't work because they are not in docs) so you can just omit the "into" part.

Reply:
this will not render well. You would need to write:

Refactor code within :func:`mne.beamformer.make_dics` and :func:`mne.beamformer.make_lcmv` into _compute_beamformer by `Britta Westner`_

now this being said we don't document refactoring as it's not helpful for users that want to learn about API changes and bug fixes.

Reply:
well ... it does *not* change any output - so I was hesitant to do a ``what's new`` until I got told to :smile:
Anyway, I am happy to either incorporate the changes or roll them back.
I will with both wait until we decide on the warning issues, since I guess there will be rebasing needed then anyway!